lib/internal/crypto/random.js

```diff
@@ -124,6 +125,12 @@ function randomFill(buf, offset, size, callback) {
 // e.g.: Buffer.from("ff".repeat(6), "hex").readUIntBE(0, 6);
 const RAND_MAX = 0xFFFF_FFFF_FFFF;
 
+// Cache random data to use in randomInt.
+const randomCache = new FastBuffer(3072);
```

Comment:
nit: it's worth to leave a comment mentioning that the buffer size here should be evenly divisible by 6.